internal/provider/resource_gitlab_label.go

```diff
@@ -43,6 +44,12 @@ var _ = registerResource("gitlab_label", func() *schema.Resource {
 				Type:        schema.TypeString,
 				Optional:    true,
 			},
+			"priority": {
+				Description:      "The priority of the label. Must be greater or equal than zero or null to remove the priority.",
```

Comment:
Did you test if it is possible to remove the priority once it has been set?

Reply:
I've added a test case :)

Reply:
The new test is comparing the value to `0`, which would be a non-null priority of value 0